api/reqs/admin.py

```diff
@@ -40,6 +42,15 @@ class PolicyAdmin(EReqsVersionAdmin):
         'workflow_phase',
     ]
 
+    def response_post_save_change(self, request, obj):
```

Comment:
Ah. This is the piece that I wasn't sure about when I was looking at this today.

Reply:
That and how to use templatetags.